app/src/main/java/net/squanchy/schedule/SchedulePageView.java

```diff
@@ -77,12 +83,21 @@ public void onAttachedToWindow() {
         subscription = service.schedule()
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe(schedule -> updateWith(schedule, event -> navigate.toEventDetails(event.id())));
+
+        proxSubscription = proximityService.observeProximityEvents()
```

Comment:
this is fine, another way of doing it is to change `subscriptions` to be `CompositeDisposable` and add the various subscriptions to that. That way you only need to invoke `clear` to clear all the subscriptions at once :) (not a blocker though)